common/app/templates/inlineJS/nonBlocking/emailIframeTracking.scala.js

```diff
@@ -0,0 +1,30 @@
+@()
+
+const sendEvent = (payload, eventType) => {
+    const msg = {
+        id: 'xxxxxxxxxx'.replace(/x/g, () =>
+            // eslint-disable-next-line no-bitwise
+            ((Math.random() * 36) | 0).toString(36)
+        ),
+        type: `ophan-iframe-${eventType}`,
+        iframeId: window.frameElement.id,
```

Comment:
fun story, `window.frameElement` doesn't work with the current apps template. Not sure how we'll work around that. Hopefully the apps-rendered template should work as expected though.